src/app/screens/Library/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react';
+import {Text, View, Image} from 'react-native';
+import styles from './styles';
+
+export interface Props {
+  title: string;
+  author: string;
+  uri: object;
+}
+
+class Library extends Component<Props> {
```

Comment:
we don't use components anymore and instead use arrow functions so we can use hooks. Please change this

Reply:
Done